releasenotes/notes/profiling-uwsgi-skip-atexit-2d931caef96837b1.yaml

```diff
@@ -0,0 +1,7 @@
+---
+other:
+  - |
+    profiling: Update docs and tests for uwsgi ``--lazy-apps`` config. When this
```

Comment:
Since this applies to all the products, does the `profiling` note here need to be more general?

Reply:
Done